package.json

```diff
@@ -21,18 +21,26 @@
     "bignumber.js": "^9.0.0"
   },
   "devDependencies": {
+    "@babel/cli": "^7.19.3",
+    "@babel/core": "^7.19.6",
+    "@babel/polyfill": "^7.12.1",
+    "@babel/preset-env": "^7.19.4",
+    "@babel/preset-typescript": "^7.18.6",
+    "@babel/register": "^7.18.9",
+    "@goodeggs/toolkit": "^7.0.1",
+    "@goodeggs/tsconfig": "^2.0.2",
     "chai": "^4.2.0",
-    "coffee-script": ">=1.8.x",
     "mocha": "^7.1.0"
   },
   "publishConfig": {
     "registry": "https://registry.npmjs.org/",
     "always-auth": true
   },
   "scripts": {
-    "compile": "coffee --bare --compile --output lib/ src/",
-    "prepublish": "npm run compile",
```

Comment:
We do need something like this, since we're using babel to build now.
```
"prepublishOnly": "yarn run build",
"build": "babel ...something...".
```

See https://github.com/goodeggs/crashpad/blob/8fc1fd8eac650d523e1d5651fdf4cfaa569c5dc9/package.json#L29-L30.

And don't forget to add the types building step here in the TS PR.